### PR TITLE
chore: move investor-foxy into lib

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Move `investor-foxy` package into lib.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

Theoretically small - if webpack builds, and the all works locally, we should be good.

## Testing

- webpack should build as expected
- the app, particularly in relation to `investor-foxy`, should work just as it did before

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4505** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4505" target="_blank">Graphite</a> 
    * **PR #4506** <a href="https://app.graphite.dev/github/pr/shapeshift/web/4506" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4506?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -177,6 +176,7 @@
     "react-table": "^7.8.0",
     "react-virtualized-auto-sizer": "^1.0.5",
     "react-window": "^1.8.7",
+    "readline-sync": "^1.4.10",
```
Comment (woodenfurniture) [inline package.json:179]: looks like this is only used by the foxy cli so maybe we should make it a dev dep

**Review by woodenfurniture — CHANGES_REQUESTED** [review]
1 change requested regarding dev dependency

File: package.json (hunk 1 above)
Comment (0xApotheosis) [reply package.json:179]: Ooh, right you are.

Comment (0xApotheosis) [reply package.json]: Moved to dev dep in https://github.com/shapeshift/web/pull/4506/commits/326089e07d24e7c56a811f20a889b99be40a6d44

**Review by woodenfurniture — APPROVED** [review]

**Comment by 0xApotheosis** [comment]
@0xApotheosis queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4506).

**Comment by 0xApotheosis** [comment]
@0xApotheosis merged this pull request with [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/4506).

**Pull request merged**